scripts/logextract.py

```diff
@@ -562,7 +569,9 @@ def finalize(self):
         out.sort()
         out = [i[2] for i in out]
         f = open(self.filename, 'wb')
```

Comment:
this could use a `with open(self.filename, 'wb') as f:` as well. (don't forget to remove the `f.close()` call at the bottom).